geode-benchmarks/src/main/java/org/apache/geode/benchmark/tasks/ExecuteFilteredFunction.java

```diff
@@ -55,13 +60,18 @@ public void setUp(BenchmarkConfiguration cfg) throws Exception {
   public boolean test(Map<Object, Object> ctx) {
     final Set<Long> filterSet = Collections.singleton(keyRange.random());
     @SuppressWarnings("unchecked")
-    final ResultCollector<?, ?> resultCollector = FunctionService
+    final Object result = FunctionService
         .onRegion(region)
         .withFilter(filterSet)
-        .execute(function);
-    resultCollector.getResult();
-    return true;
+        .execute(function)
+        .getResult();
 
+    if (isValidationEnabled) {
```

Comment:
Is it really worth an extra flag here to turn off validation? The cost of a null check and instanceof check seems inconsequential compared to a function execution, I think we could just always validate.

Reply:
The JIT can optimize out the entire block because the boolean is immutable. The null check chan't be optimized out and is at best inlined. Yes it is small. It was more important for me to just have consistency across the benchmarks that if they had any validation that it be toggle able.